Cargo.lock

```diff
@@ -1,3 +1,5 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Did you add this manually or is it the latest version of cargo that added that?

Reply:
Cargo did it. I can remove this commit if you like.

`git commit -a` betrayed me 😞

Reply:
No this is fine, I asked out of curiosity